spi/control-plane/control-plane-spi/src/main/java/org/eclipse/edc/connector/controlplane/services/spi/policydefinition/PolicyService.java

```diff
@@ -0,0 +1,28 @@
+/*
+ *  Copyright (c) 2024 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Bayerische Motoren Werke Aktiengesellschaft (BMW AG) - initial API and implementation
+ *
+ */
+
+package org.eclipse.edc.connector.controlplane.services.spi.policydefinition;
+
+import org.eclipse.edc.policy.model.Policy;
+import org.eclipse.edc.runtime.metamodel.annotation.ExtensionPoint;
+import org.eclipse.edc.spi.result.Result;
+
+/**
+ * Service that permits actions on Policy entity, e.g. validation
+ */
+@ExtensionPoint
+public interface PolicyService {
```

Comment:
Do we need this service? It doesn't seem to really do very much.

Reply:
It will host also the evaluation plan method in the next PR. The main purpose is to not expose at API level the `PolicyEngine`.  If it's not an issue i can chop the service and use directly the `PolicyEngine`

Reply:
If it will contain other functionality, then I would keep it. If it is just a wrapper, I would 🪓 it.

Reply:
I would put it into the `PolicyDefinitionService`